awscli

```diff
@@ -12,16 +12,16 @@ java "-Duser.home=${HOME}" \
 # Assume role if specified
 if [ -n "${AWS_ASSUME_ROLE_ARN:-}" ]; then
   echo -e "\nAssuming role as: ${AWS_ASSUME_ROLE_ARN}"
-  assumed_role_details=$(aws sts assume-role \
+  assumed_role_details=(`aws sts assume-role \
```

Comment:
We should never use back ticks in bash anymore. This is very old syntax and we shouldn't be encouraging the use of it.

Same goes for putting this command in an array, it can be used for the below commands but feels a bit overkill for what we are trying to achieve here.